### PR TITLE
Adding Beets Fantom deployments

### PR DESCRIPTION
**Pull request by franzns**
# Description

Adding fantom code into the subgraph but not breaking existing functionality. If we could carry these changes, that would make updating for us a lot easier and faster.

### CODE REVIEW

**Comment by franzns** [comment]
I am waiting for the test deployment to finish before opening the PR https://thegraph.com/hosted-service/subgraph/beethovenxfi/balancer-v2-test?selected=logs

**Comment by franzns** [comment]
subgraph successfully synced

File: manifest.template.yaml [hunk 1]
```diff
@@ -1240,6 +1240,39 @@ dataSources:
...
+          file: ./abis/WeightedPool.json
+        - name: ReaperLinearPoolFactory
+          file: ./abis/ReaperLinearPoolFactory.json
+        - name: LinearPool
+          file: ./abis/LinearPool.json
+        - name: ReaperLinearPool
+          file: ./abis/ReaperLinearPool.json
+      eventHandlers:
+        - event: PoolCreated(indexed address)
+          handler: handleNewReaperLinearPoolV2
```
Comment (markusbkoch) [inline manifest.template.yaml:1274]: I don't think this handler is defined

File: networks.yaml [hunk 2]
```diff
@@ -762,3 +762,89 @@ base:
...
+  TempLiquidityBootstrappingPoolFactory:
+    address: "0x2C774732c93CE393eC8125bDA49fb3737Ae6F473"
+    startBlock: 24906900
+  StablePhantomPoolFactory:
+    address: "0x0E2AFa0b5F9174750d4E84Bf9798f89b79Bd5bF4"
+    startBlock: 28658437
+  YearnLinearPoolV1:
+    address: "0x1F73AE6ED391A2b1e84Ff988A1Bb5394B78a4A71"
+    startBlock: 28657909
+  YearnLinearPoolV2:
```
Comment (markusbkoch) [inline networks.yaml:824]:
```suggestion
  YearnLinearPoolV2Factory:
```

**Review by markusbkoch — CHANGES_REQUESTED** [review]
Spotted a few typos and apparently a missing definition. Notice [reaper pools are not indexed](https://api.thegraph.com/subgraphs/name/beethovenxfi/balancer-v2-test/graphql?query=%7B%0A++pools+%28where%3A%7BpoolType%3A%22ReaperLinear%22%7D%29+%7B%0A++++id%0A++%7D%0A%7D)

**Comment by franzns** [comment]
> Spotted a few typos and apparently a missing definition. Notice [reaper pools are not indexed](https://api.thegraph.com/subgraphs/name/beethovenxfi/balancer-v2-test/graphql?query=%7B%0A++pools+%28where%3A%7BpoolType%3A%22ReaperLinear%22%7D%29+%7B%0A++++id%0A++%7D%0A%7D)

Thank you Markus, you saved me some headaches here :) Added the changes.

**Comment by markusbkoch** [comment]
> Thank you Markus, you saved me some headaches here :) Added the changes.

Would you like to redeploy for a final check before merging?

**Comment by markusbkoch** [comment]
I don't see a problem, marking it as draft until then. Thanks.

**Comment by franzns** [comment]
Subgraph completed and it looks good. Seems our "old" subgraph has one more pool but I believe that's from a factory that is not in use. Seems good to merge 👍

**Review by mendesfabio — APPROVED** [review]

**Pull request merged**